pkg/resume_backup.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright AppsCode Inc. and Contributors
+
+Licensed under the AppsCode Community License 1.0.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://github.com/appscode/licenses/raw/1.0.0/AppsCode-Community-1.0.0.md
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pkg
+
+import (
+	"context"
+	"fmt"
+
+	"stash.appscode.dev/apimachinery/apis/stash/v1beta1"
+	v1beta1_util "stash.appscode.dev/apimachinery/client/clientset/versioned/typed/stash/v1beta1/util"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+var (
+	resumeBackupExample = templates.Examples(`
+		# Pause a BackupConfigration
+		stash resume backup --namespace=<namespace> --backup-config=<backup-configuration-name>
+        stash resume backup --backup-config=asample-mongodb -n demo`)
+)
+
+func NewCmdResumeBackup() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:               "backup",
+		Short:             `resume backup`,
+		Long:              `resume backup by patching Backupconfiguration/BackupBatch`,
```

Comment:
`Resume backup by setting `paused` field of RestoreSession/RestoreBatch to `true` `